cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -296,57 +296,65 @@ The following commands are only executable against the `system` database:
+| <<administration-security-administration-dbms-privileges-privilege-management, ASSIGN PRIVILEGE>>       | DBMS                  | Determines whether the user can assign privileges using the GRANT and DENY commands.
+| <<administration-security-administration-dbms-privileges-role-management, ASSIGN ROLE>>                 | DBMS                  | Determines whether the user can grant roles.
+| <<administration-security-administration-database-constraints, CONSTRAINT MANAGEMENT>>                  | Schema                | Determines whether a user is allowed to create and drop constraints on a specific database.
+| <<administration-security-administration-database-constraints, CREATE CONSTRAINT>>                      | Schema                | Determines whether a user is allowed to create constraints on a specific database.
+| <<administration-security-administration-dbms-privileges-database-management, CREATE DATABASE>>         | DBMS                  | Determines whether the user can create new roles.
+| <<administration-security-administration-database-indexes, CREATE INDEX>>                               | Schema                | Determines whether a user is allowed to create indexes on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW NODE LABEL>>                       | Schema                | Determines whether a user is allowed to create new node labels on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>>                    | Schema                | Determines whether a user is allowed to create new property names on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW RELATIONSHIP TYPE>>                | Schema                | Determines whether a user is allowed to create new relationship types on a specific database.
+| <<administration-security-administration-dbms-privileges-role-management, CREATE ROLE>>                 | DBMS                  | Determines whether the user can create new roles.
+| <<administration-security-administration-dbms-privileges-user-management, CREATE USER>>                 | DBMS                  | Determines whether the user can create new users.
+| <<administration-security-administration-dbms-privileges-database-management, DATABASE MANAGEMENT>>     | DBMS                  | Determines whether the user can create and drop databases.
+| <<administration-security-administration-database-constraints, DROP CONSTRAINT>>                        | Schema                | Determines whether a user is allowed to drop constraints on a specific database.
+| <<administration-security-administration-dbms-privileges-database-management, DROP DATABASE>>           | DBMS                  | Determines whether the user can delete databases.
```

Comment:
The `DATABASE MANAGEMENT` says it can create and drop databases, the `DROP DATABASE` says it can delete databases, please have delete on both